### DISCUSSION TITLE
Change DEFAULT_LOCALE after database was created

### DISCUSSION THREAD

**Discussion by rchl** [firefly-iii/firefly-iii#7599] [General]
When running Firefly for the first time, I had `DEFAULT_LOCALE` set to `equal` in the `.env` file (I'm running through docker container).

After trying out Firefly, I've realized that I can't change default (currency) locale from the UI without also changing the default UI locale. So I went back to the `.env` file, changed `DEFAULT_LOCALE` (to `nb_NO` for example) and left `DEFAULT_LANGUAGE` at `en_US` but it appears that this doesn't update `locale` that is already stored in the database (in `preferences` table).

So it appears that the only way to change it, after db was created, is to either re-create db from scratch or go through the trouble of adding phpMyAdmin or similar and editing the preference manually.

Should there be an option in the UI to set the currency locale (`DEFAULT LOCALE`) separately from the UI language?

**Comment by JC5** [comment]
The default locale and language are only used for new users without any preference. Setting them later on doesn't do much because they're not used anymore.

The Docker container only contains the locales for enabled languages. It was a choice between syncing the language with the locale, maintaining two lists, or getting lots of complaints about options with no Docker support. The choice is now the first one. Perhaps this will change in the new layout, where it will be client side.

**Reply by rchl** [reply]
Just noticed that in #5746 you said that it WILL be part of the new layout. :)
